integration/solana/debug_buffer.sol

```diff
@@ -0,0 +1,150 @@
+import 'solana';
```

Comment:
The debug buffer is a substrate term. It does not make any sense for Solana. How about calling this file runtime_errors (both sol and .spec.ts file)?